Assets/Scripts/Player/HouseInputManager.cs

```diff
@@ -191,8 +194,13 @@ private void Update()
 				isSelectingPlaceable);
 		}
 
-		if (isDraggingCamera && SelectedPlaceable && !SelectedPlaceable.RotationWheel.IsRotating)
+		if (isDraggingCamera)
 		{
+			// this check must be inside the if statement, because we want the camera to rotate if:
```

Comment:
Unity provides some automatic null checks by overloading the `==` operator - if a MonoBehaviour is null, an `if (MonoBehaviour)` will return false. This works the other way too - if the MonoBehaviour is not null, a `if (MonoBehaviour)` will return true

Reply:
This doesn't necessarily require changes now, but is a good thing to know.